README.md

```diff
@@ -187,20 +197,59 @@ bazel run //validator
 ```
 **Congratulations, you are now running Ethereum 2.0 Phase 0!**
 
+## Setting up a local ETH2 development chain
+
+This section outlines the process of setting up Prysm for local interop testing with other Ethereum 2.0 client implementations. See the [INTEROP.md](https://github.com/prysmaticlabs/prysm/blob/master/INTEROP.md) file for advanced configuration options. For more background information on interoperability development, see [this blog post](https://blog.ethereum.org/2019/09/19/eth2-interop-in-review/).
+
+### Installation and Dependencies
+
+To begin setting up a local ETH2 development chain, follow the **Bazel** instructions found in the [dependencies](#dependencies) and [installation](#installation) sections respectively. After completion, continue on to [generating a genesis state](#generating-a-genesis-state).
+
+### Genesis parameters
```

Comment:
Let's get rid of this from line 208-213, we're not generating a genesis state for these instructions, we're just running a local beacon chain + validator